src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -145,14 +146,16 @@ Okta, Azure, and OneLogin have New Relic apps for both the original user model a
 
 **Recommended**: Download the full list of existing original user model users before choosing to import users. This will be a useful resource and serve as a backup, if you need it. 
 
-After downloading your original user model users, you can upload all users or just some of them. This step will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
+After downloading your original user model users, you can upload all users or just some of them. We recommend reviewing this list of users and removing any users you don't want migrated (for example, people who no longer work at your organization). 
```

Comment:
@zuluecho9, as you did above, maybe you should remind them not to migrate all the admins if they need someone left behind to manage the users on the old model?